packages/nimble-components/src/table-column/base/index.ts

```diff
@@ -81,18 +76,11 @@ export abstract class TableColumn<
 
     /**
      * @internal
-     *
-     * The template to use to render the cell content for the column
+     * The tag (element name) of the custom element that renders the cell content for the column.
+     * Should derive from TableCellView<TCellRecord, TColumnConfig>.
```

Comment:
I think this line should be a little more explicit about what "should derive from TableCellView" since it isn't the string.

One suggestion could be:
```suggestion
     * That element should derive from TableCellView<TCellRecord, TColumnConfig>.
```

Reply:
Updated